packages/bot-web-ui/src/app/__tests__/app-main.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import { initSurvicate, initSurvicateCalled } from '../../public-path';
+import React from 'react';
+import moment from 'moment';
+import { mockStore } from '@deriv/stores';
+import { render } from '@testing-library/react';
+import { TRootStore } from 'Types';
+import App from '../app-main';
+
+const root_store = {
+    ...mockStore({}),
+    common: {
+        ...mockStore({}).common,
+        server_time: moment(new Date()).utc(),
+    },
+    client: {
+        ...mockStore({}).client,
+        is_landing_company_loaded: true,
+        is_logged_in: false,
+    },
+};
+const mock_ws = {
```

Comment:
@shafin-deriv , where you doing something similar in your PR, should be have this at a common place ?

Reply:
@vinu-deriv Yes Bhai. In this PR https://github.com/binary-com/deriv-app/pull/9583 I separated the WS in another file. We can follow this to reduce duplicate code.

Reply:
done ✅